### DISCUSSION TITLE
Adding a video display for CameraExample

### DISCUSSION THREAD

**Discussion by Revan1985** [ngraziano/SharpRTSP#105] [Ideas]
Hi @ngraziano,
since I would like to improve the examples, and provide a display for the rtsp stream, I would like to add the ffmpeg for the h264/5 decoding.
Otherwise, I have seen that ffmpeg use a GPL/LGPL (depending on which version you use).
LGPL and MIT can be mixed, if I read it well, if untouching lpgl libs (would be a nuget).

You think this could be a good addition to the example?

**Comment by ngraziano** [comment]
Hi,
I prefer not to take risk with license problem in the main repository. The licence of ffmpeg is complicated when you compile with or without some module.
Can you create an other repository with the exemple (you can put the license of this repository in GPL or LGPL if you want).
I will refer your repository in the README if you want.

**Reply by Revan1985** [reply]
ok, thank you for the answer.
I will create a new repo and give the link here.
